From 86a4c47fb645ef8895557c52b6ef67c7091ad6e9 Mon Sep 17 00:00:00 2001
From: Stefano Karapetsas <stefano@karapetsas.com>
Date: Tue, 30 Apr 2013 12:56:26 +0000
Subject: Fix data passed to disabled_timeout_callback

In mate-user-share/src/user_share.c, the wrong variable is being passed as
a gsettings pointer to the disabled_timeout_callback.
https://bugs.launchpad.net/linuxmint/+bug/1005649/comments/36
---
diff --git a/src/user_share.c b/src/user_share.c
index 35e4a27..53ceec1 100644
--- a/src/user_share.c
+++ b/src/user_share.c
@@ -346,7 +346,7 @@ file_sharing_enabled_changed (GSettings *settings, gchar *key, gpointer data)
 		http_down ();
 		disabled_timeout_tag = g_timeout_add_seconds (3,
 							      (GSourceFunc)disabled_timeout_callback,
-							      client);
+							      settings);
 	}
 }
 
--
cgit 
